packages/connect/src/api/binance/api/binanceAbstractMethods.ts

```diff
@@ -0,0 +1,21 @@
+import { Capability } from '@trezor/protobuf/src/messages-schema';
+import { AbstractMethod, Payload } from '../../../core/AbstractMethod';
+import { CallMethodPayload } from '../../../events';
+import { getFirmwareRange } from '../../common/paramsValidator';
+import { getMiscNetwork } from '../../../data/coinInfo';
+
+export abstract class BinanceAbstractMethod<
+    Name extends CallMethodPayload['method'],
+    Params = undefined,
+> extends AbstractMethod<Name, Params> {
+    requiredDeviceCapabilities = [Capability.Binance];
+
+    constructor(message: { id?: number; payload: Payload<Name> }) {
+        super(message);
+        this.firmwareRange = getFirmwareRange(this.name, getMiscNetwork('BNB'), this.firmwareRange);
+    }
+
+    // firmwareRangeRequirements could be defined here instead of config.ts
```

Comment:
specifically firmwareRangeRequirement for coin. basically we have now 3 levels of fw requirements 
- coin
- method
- method feature

coin could be defined in this abstract method